native/math32.go

```diff
@@ -0,0 +1,138 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// float32 version of standard library math package routines used here.
+
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"math"
+)
+
+const (
+	uvnan    = 0x7fc00000
```

Comment:
Interestingly, I think the "uv" here comes from the 64-bit unsigned integer type uvlong in Plan 9 C ("unsigned very long"). See http://golang.org/src/lib9/fmt/nan64.c